tests/test_writer.py

```diff
@@ -1859,3 +1860,83 @@ def test_object_contains_indirect_reference_to_self():
     outpage = writer.add_blank_page(width, height)
     outpage.merge_page(reader.pages[6])
     writer.append(reader)
+
+
+@pytest.mark.enable_socket()
+def test_add_outlines_on_empty_dict():
```

Comment:
This test also passes with the current code

Reply:
the copy/paste of the URL of the test file failed and reported an other file.
thanks for notice.
this should  be fixed.